apps/services/auth/admin-api/src/main.ts

```
@@ -14,4 +15,15 @@ bootstrap({
   healthCheck: {
     database: true,
   },
+  beforeServerStart: async (app) => {
+    app.use(
+      bodyParser.json({
+        verify: (req: any, res, buf) => {
+          if (buf && buf.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)